### PR TITLE
Remove $ prompts in Readme to make commands copy-pasteable

### PR DESCRIPTION
**Pull request by arunkumaraqm**

### CODE REVIEW

**Comment by miurahr** [comment]
How do you think about documents? 
https://aqtinstall.readthedocs.io/en/latest/

**Comment by arunkumaraqm** [comment]
Same problem there too. I use a double click to select the entire line and
then copy-paste and it's really irritating to get a `$: command not found`.

**Comment by pzhlkj6612** [comment]
Hi, @arunkumaraqm . You forgot https://github.com/arunkumaraqm/aqtinstall/blame/fdcc20f1054b448ba5a2e8ef65a4c81913b856d1/README.rst#L290 .

File: README.rst [hunk 1]
```diff
@@ -78,8 +78,8 @@ Same as usual, it can be installed with `pip`
 
 .. code-block:: bash
```
Comment (miurahr) [inline README.rst:79]: It would be better to change `code-block:: console` to show better syntax highlight for interactive session example

**Comment by miurahr** [comment]
@arunkumaraqm 
> Same problem there too. 

see #322

**Comment by miurahr** [comment]
@arunkumaraqm are there the same issue?
https://github.com/arunkumaraqm/aqtinstall/blame/fdcc20f1054b448ba5a2e8ef65a4c81913b856d1/README.rst#L210

**Comment by pzhlkj6612** [comment]
Hi, @miurahr .

> are there the same issue?
> https://github.com/arunkumaraqm/aqtinstall/blame/fdcc20f1054b448ba5a2e8ef65a4c81913b856d1/README.rst#L210

Those CMD blocks don't have any $ prompt. In my opinion, it would be better to be modified in another PR since the title of this PR is "remove **$ prompts** in Readme".

**Comment by arunkumaraqm** [comment]
@pzhlkj6612  Corrected the $ that slipped through.

**Comment by miurahr** [comment]
Thank you.

**Comment by miurahr** [comment]
Now change to use `console` for lexer. On github, you can use GUI tool to copy example.

**Pull request merged**